contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -166,13 +168,17 @@ In this case, only the `root` component can be accessed in Vue while the rest ar
       template: {
         <button v-on:click="$emit('increased-count', count+1)">Increment item count</button>
       }
-    }
+    })
 
     /* Inside todo-list component */
     template: {
-        v-on:increased-count="updateCount"
+      v-on:increased-count="updateCount"
```

Comment:
is this an old syntax? The closest I could find in the docs was using `on:` in render functions.
Could fix this as well if so